src/common/github/pull-request-test-workflow.ts

```diff
@@ -101,6 +108,19 @@ export class PullRequestTest extends Component {
         ]),
       })
     }
+
+    if (options.isPlaywrightEnabled) {
```

Comment:
@bdoof do you think we can refactor the code in a more modular way in one of the following PRs? The conditions in the code don't look great to me.

By modules I mean different GitHub projects, instantiated like this:
```typescript
new GitHub.LightHouseWorkflow(...)
new GitHub.PlaywrightWorkflow(...)
```

Reply:
No, a `projen` project owns `package.json` and the latter can't be shared with other projects. Though you can set up a nested structure and has the e2e tests as a separate project.